.travis.yml

```diff
@@ -9,8 +9,8 @@ install:
   - pip install -r requirements-dev.txt
   - pip install -e .
 script:
-  - flake8 unpfayep
-  - py.test --ds=project.settings --verbose --cov unpfayep unpfayep
+  - flake8 tuneme
```

Comment:
This should be able to be just `flake8` without `tuneme` but that may be a scaffolding problem.